internal_transfers/actions/src/imbalance.ts

```diff
@@ -15,8 +15,8 @@ export function aggregateTransfers(
 ): ImbalanceMap {
   let accumulator: ImbalanceMap = new Map<string, bigint>();
   transfers.map((transfer) => {
-    const { to, from, amount, token } = transfer;
-
+    let { to, from, amount, token } = transfer;
+    token = token.toLowerCase();
```

Comment:
This change is a bit ugly and is a result of having to "normalize" hex strings. We should probably find a global/universal solution to this problem instead of having to make conversions all the time.

Reply:
Created an issue to handle this:

https://github.com/cowprotocol/solver-rewards/issues/236